package.json

```diff
@@ -1,23 +1,46 @@
 {
   "name": "@mongodb-js/zstd",
-  "version": "1.2.2",
+  "version": "2.0.0-beta.0",
   "main": "index.js",
   "types": "index.d.ts",
   "repository": "https://github.com/mongodb-js/zstd",
   "files": [
     "index.d.ts",
     "index.js"
   ],
+  "dependencies": {
+    "bindings": "^1.5.0",
```

Comment:
I am surprised we're adding bindings here, don't we have a ticket to stop using this?

Reply:
I think it's unfortunate if we release with the known issues coming from that package. We can merge this, but do you think it's possible we can ship the final 2.0.0 without this? We can follow up

Reply:
No, I don't think it is likely that we can complete that with all the other work we need to do in the next 8 work days (our timeline for this project).  I am prioritizing consistency over a full overhaul/modernization of this package.

Reply:
My understanding is that we need to enumerate the possible file paths of the .node file so it doesn't seem like an overhaul. Meanwhile, we're potentially introducing a regression in our major release of this package since we didn't used to use this.

We likely could adapt the code in https://github.com/mongodb-js/zstd/blob/e7e8df676e3a6948b67708938495eb876d5bf117/bindings.js to just have a few different strings and it would do the work we need. 

We can revisit trying to include this in the 2.0, doesn't need to hold up this PR.